packages/amplify-category-api/src/graphql-transformer/transform-graphql-schema-v2.ts

```diff
@@ -273,17 +273,13 @@ export const getUserOverridenSlots = (userDefinedSlots: Record<string, UserDefin
     .filter((slotName) => slotName !== undefined);
 
 const getRDSLayerMapping = async (): Promise<RDSLayerMapping> => {
-  try {
-    const response = await fetch(LAYER_MAPPING_URL);
-    if (response.status === 200) {
-      const result = await response.json();
-      return result as RDSLayerMapping;
-    }
-  } catch (err) {
-    // Ignore the error and return default layer mapping
+  const response = await fetch(LAYER_MAPPING_URL);
+  if (response.status === 200) {
+    const result = await response.json();
+    return result as RDSLayerMapping;
+  } else {
+    throw new Error(`Unable to retrieve layer mapping from ${LAYER_MAPPING_URL} with status code ${response.status}.`);
```

Comment:
We could add a re-try after a pause here to filter out any transient network connection issues. Can be a backlog item.